x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/utils.ts

```diff
@@ -51,22 +59,46 @@ export const newTrustedAppItemToExceptionItem = ({
   entries,
   name,
   description = '',
-}: NewTrustedApp): NewExecptionItem => {
+}: NewTrustedApp): NewExceptionItem => {
   return {
     _tags: tagsListFromOs(os),
     comments: [],
     description,
-    entries,
+    // @ts-ignore
```

Comment:
What's the error here?

Reply:
@kevinlog ,
It has to do with the fact that `entries` in Trusted Apps is an Array of different types base on the outer value for `os` and in order to get around it, we would have to do (needless) runtime castings using type guards. In this instance, i think it's ok to ignore it, but open to suggestions.

Error is:
![image](https://user-images.githubusercontent.com/56442535/93817007-4a13f480-fc26-11ea-9ae2-d8983194d7b1.png)